packages/jest-snapshot/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jest-snapshot",
+  "version": "12.0.2",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "src/index.js",
+  "dependencies": {
+    "jest-util": "^12.0.2",
+    "react": "^15.0.2",
```

Comment:
I'm worried about making react a dependency of this. It's probably just going to cause trouble. I believe that snapshots should not be opinionated and use some sort of duck-typing to figure out what it should do. I'll have to think about this.